### PR TITLE
changed all arrow functions that I could

### PR DESCRIPTION
**Pull request by jfkinslow**
Changed all arrow functions possible to use the shorthand syntax

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -75,32 +75,28 @@ console.log("The last car is a %s %s", inventory[lastCar].car_make, inventory[la
...
-let oldCars = carYears.filter((current) => {
-    return current < 2000;
-});
+let oldCars = carYears.filter((current) => current < 2000);
 console.log(oldCars.length); 
 
 // ==== Challenge 6 ====
 // A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
-let BMWAndAudi = inventory.map((current, index, arr) => {return (current.car_make == "Audi" || current.car_make == "BMW") ? current : undefined });
+let BMWAndAudi = inventory.filter((current) => (current.car_make == "Audi" || current.car_make == "BMW"));
```
Comment (Dilstom) [inline assignments/arrays.js]: good job with this assignment! (it states that You should Only use a basic for loop, but I do understand that you have a deep understanding of a for a loop already ;) )

File: assignments/function-conversion.js [hunk 2]
```diff
@@ -1,30 +1,20 @@
...
+console.log(subtract(1,2));
 
 
 // Stretch
 
 exampleArray = [1,2,3,4];
-const triple = exampleArray.map((num) => {
-  return num * 3;
-});
+const triple = exampleArray.map((num) => num * 3);
```
Comment (Dilstom) [inline assignments/function-conversion.js]: perfect!

File: assignments/objects.js [hunk 3]
```diff
@@ -84,7 +80,7 @@ console.log(interns.Antonietta.gender);
 console.log(interns.Kennan.speak());
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
-console.log(interns.Antonietta.multiplyNums(3,4));
+console.log(interns.Antonietta.multiplyNums(7,2));
```
Comment (Dilstom) [inline assignments/objects.js]: very nice!

File: assignments/arrays.js (hunk 1 above)
Comment (jfkinslow) [reply assignments/arrays.js]: that was a result of the stretch goal to use array methods. i just tried to take and refactor my old code.

**Pull request merged**